### DISCUSSION TITLE
Needs clarification on sunbird use case and how to add that scenario in ODK form

### DISCUSSION THREAD

**Discussion by Nik720** [Samagra-Development/SamagraX-community#20] [Troubleshooting and Clarifications]
@SaketSinha @aayushkedia95

In the sunbird bot, we have use cases with different links for portal bot and mobile bot. How we can add both the links or different messages in odk form for both portal and mobile?

what we thought is 
1) If there is a way in odk form that we can have two different columns for labeling and based on user platform (portal or mobile) bot will show that label
2) we can add an object of two different messages in a label column and create a new transformer or adapter where we can decide which message needs to show

Can you please help us with this use case and provide your suggestions or if you have a better way to tackle this problem, please let us know

cc - @vinukumar-vs @vrayulu

**Comment by aayushkedia95** [comment]
@Nik720, since both bots (WA and Web) are different, as discussed, in the current flow both will have different ODK forms. And so according will have to be maintained. 

However, this can be handled by creating a new transformer.
